common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/BlockPushingListener.java

```diff
@@ -25,6 +25,8 @@
  * code reuse for handling block push and fetch retry.
  */
 public interface BlockPushingListener extends BlockTransferListener  {
```

Comment:
Nit: Are we keeping `BlockPushingListener`? My suggestion was to call it `BlockPushListener`.

Reply:
I think `BlockPushingListener` is more consistent with `BlockFetchingListener.

Reply:
Yeah, but `BlockPushListener` sounds better and is consistent with `BlockTransferListener`. Not a big deal though.

Reply:
`BlockPushingListener` is supposed to be a peer to `BlockFetchingListener`. Their names being consistent feels more appropriate.